node/src/components/small_network/tasks.rs

```diff
@@ -63,6 +63,13 @@ use crate::{
 /// successfully handed over to the kernel for sending.
 pub(super) type MessageQueueItem<P> = (Arc<Message<P>>, Option<AutoClosingResponder<()>>);
 
+struct HandshakeOutcome {
+    framed_transport: FramedTransport,
+    public_addr: SocketAddr,
+    peer_consensus_public_key: Option<PublicKey>,
+    is_joiner: bool,
```

Comment:
```suggestion
    is_peer_joiner: bool,
```

I prefer this over `is_joiner` as the latter can be confused with a flag indicating whether _we_ are a joiner node.

Reply:
I agree.
Changed back to `is_peer_joiner` in 96d9555915bfc004396edba41cf9bc4d11a98061